src/main/fc/init.c

```diff
@@ -999,5 +999,11 @@ void init(void)
 
     tasksInit();
 
+#ifdef SIMULATOR_MULTITHREAD
+    rescheduleTask(TASK_SERIAL, 1);
+    rescheduleTask(TASK_RX, 1);
+    pinioBoxTaskControl();
```

Comment:
Move pinoBoxTaskControl here, otherwise, when we set arming stick, the SITL will crash.